e2e/test/scenarios/dashboard/reproductions/33637-xray-should-not-have-download-results-edit.cy.spec.js

```diff
@@ -0,0 +1,33 @@
+import { main, navigationSidebar, restore } from "e2e/support/helpers";
```

Comment:
I will remove this test before merging, for now I'll keep to double check if i break something when changing things